packages/grid/x-grid/src/tests/apiRef.XGrid.test.tsx

```diff
@@ -0,0 +1,77 @@
+import { ApiRef, useApiRef } from '@material-ui/data-grid';
+import { XGrid } from '@material-ui/x-grid/XGrid';
+import { expect } from 'chai';
+import * as React from 'react';
+import { getColumnValues, sleep } from 'test/utils/helperFn';
+import { render } from '@testing-library/react';
```

Comment:
All imports to `@testing-library/react` are prohibited, we proxy them all with `test/utils`.

```suggestion
import { createClientRenderStrictMode } from 'test/utils';
```

x all the other places

Reply:
it doesn't work if I use the wrapper

Reply:
Did you try disabling strict mode? Maybe the logic isn't idiomatic to React and break in strict mode, there is a precedent for it in https://github.com/mui-org/material-ui-x/pull/723/files#diff-7496f3f0a0d951e1221c1689081b949af7cc911ddf81d37b1b826c86a6a88a5dR52.